lib/ra10ke/dependencies.rb

```diff
@@ -138,6 +138,43 @@ def print_table(mods)
     end
   end
 
+  def define_task_convert_to_git(*_args)
+    desc "Print outdated forge modules"
+    task :convert_to_git do
+      require 'ra10ke/git_repo'
+      require 'r10k/puppetfile'
+      require 'puppet_forge'
+
+      PuppetForge.user_agent = "ra10ke/#{Ra10ke::VERSION}"
+      puppetfile = get_puppetfile
+      puppetfile.load!
+      PuppetForge.host = puppetfile.forge if puppetfile.forge =~ /^http/
+
+      # ignore file allows for "don't tell me about this"
+      ignore_modules = []
+      if File.exist?('.r10kignore')
+        ignore_modules = File.readlines('.r10kignore').each {|l| l.chomp!}
```

Comment:
that file would be a list of modules that should not be converted? Is that file already used somewhere else? Otherwise it should be documented.

Reply:
Yeah I just copied it from the task below this one.  I assumed it was a ra10ke feature.  Not something I added but just continued to support.

Reply:
Interesting, never saw it previously :D